tests/integration/security/main_test.go

```diff
@@ -45,10 +45,30 @@ func setupConfig(ctx resource.Context, cfg *istio.Config) {
 	if cfg == nil {
 		return
 	}
-
+	// command to generate certificate
+	// https://github.com/istio/istio/blob/master/samples/jwt-server/testdata/README.MD
```

Comment:
Is this using the `ca.crt`? Better to clarify which cert is used since the REAME generates two certs.

Reply:
oh it's server.crt. I'll add more comments here.